.env.example

```
@@ -33,3 +33,7 @@ TURNSTILE_SITE_KEY=1x00000000000000000000AA
 
 # ── CORS ─────────────────────────────────────────────
 CORS_ALLOW_ALL=false
+
+# ── MeiliSearch ─────────────────────────────────────
+MEILI_URL=https://search.matrictrend.app
+MEILI_MASTER_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MEILI_MASTER_KEY key should go before the MEILI_URL key

(UnorderedKey)